### PR TITLE
using the new contracts package version

### PR DESCRIPTION
**Pull request by aviled**

### CODE REVIEW

**Comment by aviled** [comment]
> namespace Microsoft.Azure.Monitoring.SmartDetectors.Clients

I'll consult with Netta and open a task

---
In reply to: [376423800](https://github.com/liadtal/Smart-Alerts/pull/115#issuecomment-376423800) [](ancestors = 376423800)

---
Refers to: src/SDK/SmartDetectorsShared/Clients/QueryRunInfoProvider.cs:7 in 7bc9b15. [](commit_id = 7bc9b155abae73ca2ca68299437c352246ddf806, deletion_comment = False)

File: src/management/server/SmartDetectorsManagementApi/EndpointsLogic/SmartDetectorApi.cs [hunk 1]
```diff
@@ -55,7 +53,7 @@ public SmartDetectorApi(ISmartDetectorRepository smartDetectorRepository)
                    Id = manifest.Id,
                    Name = manifest.Name,
                    SupportedCadences = new List<int>(manifest.SupportedCadencesInMinutes),
-                   SupportedResourceTypes = new List<ResourceType>(manifest.SupportedResourceTypes),
+                   SupportedResourceTypes = manifest.SupportedResourceTypes.Select(resourceType => (ResourceType)resourceType).ToList(),
```
Comment (morkra) [inline src/management/server/SmartDetectorsManagementApi/EndpointsLogic/SmartDetectorApi.cs]: >(resourceType => (ResourceType)resourceType [](start = 82, length = 43)

sooooo ugly ):

Comment (morkra) [reply src/management/server/SmartDetectorsManagementApi/EndpointsLogic/SmartDetectorApi.cs]: Does it work?

---
In reply to: [177330532](https://github.com/liadtal/Smart-Alerts/pull/115#discussion_r177330532) [](ancestors = 177330532)

**Comment by aviled** [comment]
> namespace Microsoft.Azure.Monitoring.SmartDetectors.Clients

let's wait for the new name because "Smart" is going to change

---
In reply to: [376423800](https://github.com/liadtal/Smart-Alerts/pull/115#issuecomment-376423800) [](ancestors = 376423800)

---
Refers to: src/SDK/SmartDetectorsShared/Clients/QueryRunInfoProvider.cs:7 in 7bc9b15. [](commit_id = 7bc9b155abae73ca2ca68299437c352246ddf806, deletion_comment = False)

**Comment by aviled** [comment]
> namespace Microsoft.Azure.Monitoring.SmartDetectors.Clients

opened 1219172

---
In reply to: [376423800](https://github.com/liadtal/Smart-Alerts/pull/115#issuecomment-376423800) [](ancestors = 376423800)

---
Refers to: src/SDK/SmartDetectorsShared/Clients/QueryRunInfoProvider.cs:7 in 7bc9b15. [](commit_id = 7bc9b155abae73ca2ca68299437c352246ddf806, deletion_comment = False)

File: src/management/server/SmartDetectorsManagementApi/EndpointsLogic/SmartDetectorApi.cs (hunk 1 above)
Comment (aviled) [reply src/management/server/SmartDetectorsManagementApi/EndpointsLogic/SmartDetectorApi.cs]: yep.
added comment.

---
In reply to: [177330666](https://github.com/liadtal/Smart-Alerts/pull/115#discussion_r177330666) [](ancestors = 177330666,177330532)

**Pull request merged**